src/dlgSourceEditorArea.cpp

```diff
@@ -20,14 +20,55 @@
 
 
 #include "dlgSourceEditorArea.h"
+#include "edbee/edbee.h"
+#include "edbee/texteditorwidget.h"
+#include "edbee/models/textdocument.h"
+#include "edbee/views/textrenderer.h"
+#include "edbee/views/texttheme.h"
+#include "edbee/models/textgrammar.h"
+#include "edbee/models/texteditorconfig.h"
 
-#include "THighlighter.h"
+#include <QDir>
+
+
+//#include "THighlighter.h"
 
 
 dlgSourceEditorArea::dlgSourceEditorArea(QWidget* pF) : QWidget(pF)
 {
     // init generated dialog
     setupUi(this);
-    highlighter = new THighlighter(editor->document());
-    editor->setTabStopWidth(25);
+
+    //highlighter = new THighlighter(editor->document());
+    //editor->setTabStopWidth(25);
+
+    // Setting up the edbee widget
+
+    edbee::Edbee* ei = edbee::Edbee::instance();
+
+    ei->autoInit();
+    ei->autoShutDownOnAppExit();
+
+    ei->setGrammarPath(  "G:\\Temp\\edbee\\syntaxfiles\\" );
+    ei->setThemePath( "G:\\Temp\\edbee\\themes\\" );
```

Comment:
Line 52 and 53 should have been removed. The configured paths aren't used since we only need one syntax file and one theme, and theyre referenced directly.